test/test_jsinterp_parse.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from __future__ import unicode_literals
+
+import os
+import sys
+import logging
+import copy
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from youtube_dl.jsinterp import JSInterpreter
+from .jstests import gettestcases
+
+
+def traverse(node, tree_types=(list, tuple)):
+    if type(node) == zip:
```

Comment:
```isinstance(node, zip)```

I'm afraid this line is fragile as Python spec does not require ```zip``` as a class. Of course it's fine if there's no other solutions.